src/pynwb/data/nwb.file.yaml

```diff
@@ -140,78 +141,76 @@ groups:
     datasets:
     - name: data_collection
       dtype: text
-      doc: 'Notes about data collection and analysis.COMMENT: Can be from Methods'
+      doc: Notes about data collection and analysis.
       quantity: '?'
     - name: experiment_description
       dtype: text
-      doc: 'General description of the experiment.COMMENT: Can be from Methods'
+      doc: General description of the experiment.
       quantity: '?'
     - name: experimenter
       dtype: text
-      doc: 'Name of person who performed the experiment.COMMENT: More than one person
-        OK. Can specify roles of different people involved.'
+      doc: Name of person(s) who performed the experiment. Can also specify roles
```

Comment:
We might want to change this to be a list in order to list multiple experimenters

Reply:
Agreed, but that is a separate issue. I did not want to pack too many changes into this PR.